Week2/Homework/app.js

```diff
@@ -0,0 +1,97 @@
+const mysql = require('mysql');
+const prompt = require('prompt');
+const util = require('util');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'world'
+});
+
+
+connection.connect();
+
+const execQuery = util.promisify(connection.query.bind(connection));
+const input = util.promisify(prompt.get.bind(this));
+
+async function seedDatabase() {
+    let input_value = '';
+    prompt.start();
+    try {
+        console.log(`Choose from the following options:
+      1. What is the capital of country X ? (Accept X from user).
+      2. List all the languages spoken in the region Y (Accept Y from user).
+      3. Find the number of cities in which language Z is spoken(Accept Z from user).
+      4. Accept the region and language from the user:
+         Are there any countries in this region with the given language as the official language?
+         If yes, display those countries. If no, display FALSE. 
+         E.g. (A) input region: 'Western Europe' and input language: 'Dutch' output should be Belgium and Netherlands.
+         (B) input region: 'Western Europe' and input language: 'Hindi' output should be 'FALSE'
+      5. List all the continents with the number of languages spoken in each continent.
+      `);
+
+        const choices = await input(['choice']);
+        input_value = choices.choice;
+
+        switch (input_value) {
+            case '1':
+                const firstChoice = await input(['country']);
+                let country = firstChoice.country;
+                const query1 = `SELECT  city.name from country left join city on country.capital = city.id  where country.name = ?`;
```

Comment:
It works because all countryLanguage have a country code as it's the foreign key but here we should use inner join (or just join).